metadata-ingestion/tests/unit/test_snowflake_shares.py

```diff
@@ -336,13 +338,85 @@ def test_snowflake_shares_workunit_inbound_and_outbound_share(
             siblings_aspect = wu.get_aspect_of_type(Siblings)
             assert siblings_aspect is not None
             if "db1" in wu.get_urn():
+                assert not siblings_aspect.primary
                 assert len(siblings_aspect.siblings) == 1
                 assert siblings_aspect.siblings == [
                     wu.get_urn().replace("instance1.db1", "instance2.db1")
                 ]
             else:
+                assert siblings_aspect.primary
                 assert len(siblings_aspect.siblings) == 2
                 assert siblings_aspect.siblings == [
                     wu.get_urn().replace("instance1.db2", "instance2.db2_from_share"),
                     wu.get_urn().replace("instance1.db2", "instance3.db2"),
                 ]
+
+
+def test_snowflake_shares_workunit_inbound_and_outbound_share_no_platform_instance(
+    snowflake_databases: List[SnowflakeDatabase],
+) -> None:
+    config = SnowflakeV2Config(
+        account_id="abc12345",
+        shares={
+            "share1": SnowflakeShareConfig(
```

Comment:
"db1" (primary, included in share) and "db2"(secondary, created from share) are present in this account, as present in `snowflake_databases`.